requirements-dev.txt

```diff
@@ -60,23 +59,21 @@ pytest==6.2.5
     # via -r requirements-dev.in
 pyyaml==6.0
     # via pre-commit
-regex==2021.11.10
```

Comment:
why does this (and typed-ast/appdirs) need to be removed?

Reply:
I'm not actually sure, but assuming that some other requirement needed it, but it's no longer needed, the `requirements-dev.txt` is automatically generated from the `requirements-dev.in` using `pip-compile`

Reply:
@nawatts if you have time, can you confirm this is the reason?

Reply:
> assuming that some other requirement needed it, but it's no longer needed

Yes, this is the reason. `pip-compile` adds `# via <package>` comments to this file to indicate where transitive dependencies originate from. `regex` and `typed-ast` both had `# via black` comments, so it looks like the newer version doesn't require them anymore.

Reply:
Thank you @nawatts!